.github/workflows/e2e-tests.yml

```diff
@@ -168,7 +168,6 @@ jobs:
 
       - name: Tilt CI - standard E2E tests
         shell: bash
-        if: ${{ inputs.run-expensive-tests == false }}
```

Comment:
ohhh so previously on the daily cron workflow we weren't running the "non-expensive" tests? 👀

Reply:
It was but it was hidden under npm command in package.json which I changed yesterday. Instead, now it's explicit in GH workflows - every time we run standard e2e tests and daily + main branch workflows will additionally run expensive ones